packages/data-service/src/data-service.ts

```diff
@@ -105,9 +105,6 @@ import type {
 } from '@mongodb-js/devtools-connect';
 import { omit } from 'lodash';
 
-const PREVIEW_TIMEOUT = 1000;
-const PREVIEW_SAMPLE = 10;
```

Comment:
No super hard feelings about these, happy to restore them, but if we keep them I’d probably choose a different name that’s more indicative of these applying to *update* preview specifically